### ISSUE TITLE
Sounds do not respect system volume

### ISSUE THREAD

**Issue by devios1** [TUNER88/iOSSystemSoundsLibrary#1]
I'm not sure if it's because these are considered "alert sounds" and thus use the ringer volume, but adjusting my system volume has no effect on the volume these sounds play at. In fact they even play when the phone is muted (though not if the ringer is turned off). I'm just curious if there is a way to make the sounds play at the system volume instead of the alert volume.

**Comment by w-i-n-s** [comment]
Yes, you're right - no affect at all. Because these are alert sounds.
